content/quickstart/go/tracing-intermediate.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Tracing"
+title: "Tracing (Intermediate)"
```

Comment:
We might want to keep the alias either on the beginner script or this one.

Reply:
that is:
alias: [/quickstart/go/tracing]

perhaps for the tracing-beginner.md file